task1/task1.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Task 1</title>
+</head>
+
+<body>
+    <table border=1 cellspacing=0 cellpadding=5px>
+        <thead>
+            <tr height=100px>
+                <th colspan="3">
+                    <font size="6">Markup of “Basic HTML” task</font>
+                </th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr valign=top>
+                <td width=200px>
+                    <table>
+                        <thead>
+                            <tr>
+                                <th align=left>
+                                    <font size="5">Menu</font>
+                                </th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr>
+                                <td><a href="#">Main</a></td>
```

Comment:
it's look like a list. maybe it is a [list](https://www.w3schools.com/tags/tag_ul.asp)?

Reply:
Yes, but ul an ol have bullets and numbers showing up next to items and you cannot hide them without CSS, therefore I used a table for the list to appear like in mockup